src/Container.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './container.css';
+
+interface ContainerProps {
+  children: React.ReactNode;
+  maxWidth?: string;
+}
+
+const Container: React.FC<ContainerProps> = ({
+  children,
+  maxWidth = '1200px'
+}) => {
+  const containerStyle: React.CSSProperties = {
+    maxWidth,
+    margin: '0 auto',
+    padding: '0 20px'
+  };
+
+  return (
+    <div className="container" style={containerStyle}>
```

Comment:
```suggestion
  return (
    <div className="container" style={{ maxWidth, margin: '0 auto', padding: '0 20px' }}>
```

변수가 jsx에서 공유되어 사용되는건 아니라서 요런건 별도 변수 선언없이 바로 prop으로 전달해줘도 괜찮아요. 꼭 수정하진 않으셔도 됩니다.

Reply:
모든 코드 리뷰 피드백 반영하였습니다. 확인 부탁드립니다!